app/controllers/analytics_controller.rb

```diff
@@ -18,6 +18,8 @@ def index
     @cases_sorted_by_followers = Case.sorted_by_followers 10
     @most_visited_cases = DetermineVisitsToCases.call(@visits_this_week.sorted_by_hits(13))
     @most_recent_comments = Comment.sorted_by_creation 15
+    @case_updates_30 = PaperTrail::Version.where('created_at > ?', 30.days.ago)
+    @case_updaters_count_10 = @case_updates_30.group(:whodunnit).order('count_id DESC').limit(10).count(:id)
```

Comment:
Let's answer the question above before we dissect this query.

Reply:
so the case_updaters_count_10 query takes all the versions from the past 30 days and groups them by the whodunnit (id of the responsible user), returning the id and the number of versions it "has"

Reply:
This group of queries needs to be in another object that the `AnalyticsController` calls, I'm thinking.

Reply:
Honestly,  I think it might be better to have a column on the user that records the last time they created or updated a case.  Then, it's just a matter of querying for that column rather than pulling what feels like an arbitrary number of versions and running a group query on it.